core/src/com/unciv/ui/worldscreen/WorldMapHolder.kt

```diff
@@ -349,21 +369,30 @@ class WorldMapHolder(internal val worldScreen: WorldScreen, internal val tileMap
                     updateTilegroupsForSelectedUnit(unit, playerViewableTilePositions)
                 }
             }
-            unitActionOverlay != null -> {
-                unitActionOverlay!!.remove()
-                unitActionOverlay = null
+            unitActionOverlays.isNotEmpty() -> {
+                removeUnitActionOverlay()
+            }
+        }
+
+        try {
+            for (group in tileGroups[selectedTile]!!) {
+                group.showCircle(Color.WHITE)
             }
+        } catch(ex: Exception){
+            //Dont know if this can happen
```

Comment:
Remove this - if there are exceptions, we want them to bubble so we can figure out why they happen, rather than ignoring them :)